internal/pkg/plugin/helminstaller/defaults/argocd.go

```diff
@@ -1,15 +1,18 @@
-package argocd
+package defaults
 
 import (
 	"github.com/devstream-io/devstream/internal/pkg/plugininstaller/helm"
 	helmCommon "github.com/devstream-io/devstream/pkg/util/helm"
 	"github.com/devstream-io/devstream/pkg/util/types"
 )
 
-var defaultHelmConfig = helm.Options{
+var toolName = "argocd"
```

Comment:
Maybe `toolArgocd` is better

Reply:
why？

Reply:
get it. the "defaults" package will have more tools.